docs/configuration_file.md

```diff
@@ -137,6 +137,9 @@ Some options are also mandatory.
 
 * `can_orchestrate` (*optional*, `boolean`) — allows using orchestrator build, default is false
 
+* `info_url_format` (*optional*, `string`) — format for `url` dockerfile label, by default :
+`https://access.redhat.com/containers/#/{authoritative-source-url}/{name}/images/{version}-{release}`, you can use any label,
```

Comment:
I actually think we shouldn't have a default value. If the config file doesn't specify a url format we should leave that label alone.